src/app/visual/vertical-tabset/vertical-tabs-visual.component.ts

```diff
@@ -36,6 +36,8 @@ export class VerticalTabsVisualComponent {
 
   public tabDisabled: boolean = true;
 
+  public maintainTabContent: boolean = false;
```

Comment:
Check your alphabetical ordering of properties here.

Reply:
Done